reframe/core/schedulers/__init__.py

```diff
@@ -326,6 +327,18 @@ def nodelist(self):
         '''
         return self._nodelist
 
+    @property
+    def hostlist(self):
+        '''The hostlist of this job.
+
+        The value of this field is scheduler-specific.
+
+        .. versionadded:: 3.6
```

Comment:
Version strings should be conformant to version semantic syntax, i.e., this one should be 3.6.0.